### ISSUE TITLE
Issue with install on ubuntu 18.04 desktop

### ISSUE THREAD

**Issue by ghost** [me-box/databox#271]
By default, 18.04 ubuntu doesnt come with net-tools (and thus no ifconfig), this causes databox start to fail with: 

```
Error response from daemon: This node is not a swarm manager. Use "docker swarm init" or "docker swarm join" to connect this node to swarm and try again.
ERROR: Network databox-system-net declared as external, but could not be found. Please create the network manually using `docker network create databox-system-net` and try again.
"docker inspect" requires at least 1 argument.
See 'docker inspect --help'.

Usage:  docker inspect [OPTIONS] NAME|ID [NAME|ID...] [flags]

Return low-level information on Docker objects
undefined:1



SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at main (/cwd/src/createResolvConf.js:5:24)
    at Object.<anonymous> (/cwd/src/createResolvConf.js:22:1)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)
    at startup (internal/bootstrap/node.js:240:19)
Pulling app-server (databoxsystems/app-server:)...
latest: Pulling from databoxsystems/app-server
605ce1bd3f31: Pull complete
454cb03c28f3: Pull complete
d842fa9e2653: Pull complete
dc19af70ff10: Pull complete
0d0c83bbd6d3: Pull complete
cdcc23dd3deb: Pull complete
3b1a2b1a45c9: Pull complete
Digest: sha256:4639e8b7ebedd21e76f66d30f8b1be05af8971f201331630418040f5ae195c30
Status: Downloaded newer image for databoxsystems/app-server:latest
Creating databox_app-server_1 ... done
this node is not a swarm manager. Use "docker swarm init" or "docker swarm join" to connect this node to swarm and try again
[2018-07-04 11:47:09 databox-start]: ERROR: Problem with docker stack deploy
```

installing net-tools fixed this but the error message doesn't reflect the actual problem and is therefore useless.

**Comment by Toshbrown** [comment]
This will be fixed in the next version as it no longer relies on ifconfig

**Comment by Toshbrown** [comment]
fixed on 0.5.0
